CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ v4.3.0
   certain input types (#893)
 * Cache reference lookups for subschemas (#893)
 * Use cached lookups for resolving fragments if the referent document is known (#893)
+* Replace the ``Validator.evolve`` method with an equivalent class attribute
```

Comment:
This is an implementation detail, we can save mentioning it I think.

And for the other two I'd maybe just say something simpler like "resolving refs has had performance improvements" and have that cover the whole change, someone can look at the PR for details.

Reply:
Done! :)